docs/docs/flink.md

```diff
@@ -24,20 +24,20 @@ Apache Iceberg supports both [Apache Flink](https://flink.apache.org/)'s DataStr
 
 | Feature support                                             | Flink | Notes                                                                                  |
 | ----------------------------------------------------------- |-------|----------------------------------------------------------------------------------------|
-| [SQL create catalog](#creating-catalogs-and-using-catalogs) | ✔️    |                                                                                        |
-| [SQL create database](#create-database)                     | ✔️    |                                                                                        |
-| [SQL create table](#create-table)                           | ✔️    |                                                                                        |
-| [SQL create table like](#create-table-like)                 | ✔️    |                                                                                        |
-| [SQL alter table](#alter-table)                             | ✔️    | Only support altering table properties, column and partition changes are not supported |
-| [SQL drop_table](#drop-table)                               | ✔️    |                                                                                        |
-| [SQL select](#querying-with-sql)                            | ✔️    | Support both streaming and batch mode                                                  |
-| [SQL insert into](#insert-into)                             | ✔️ ️  | Support both streaming and batch mode                                                  |
-| [SQL insert overwrite](#insert-overwrite)                   | ✔️ ️  |                                                                                        |
-| [DataStream read](#reading-with-datastream)                 | ✔️ ️  |                                                                                        |
-| [DataStream append](#appending-data)                        | ✔️ ️  |                                                                                        |
-| [DataStream overwrite](#overwrite-data)                     | ✔️ ️  |                                                                                        |
-| [Metadata tables](#inspecting-tables)                       | ✔️    |                                                                                        |
-| [Rewrite files action](#rewrite-files-action)               | ✔️ ️  |                                                                                        |
+| [SQL create catalog](../flink-ddl.md#create-catalog) | ✔️    |                                                                                        |
+| [SQL create database](../flink-ddl.md#create-database)                     | ✔️    |                                                                                        |
```

Comment:
```suggestion
| [SQL create database](../flink-ddl.md#create-database) | ✔️    |                                                                                        |
```

Reply:
This will create whitespace change. Don't we want to avoid it?

Reply:
yes, but the formatting of the table looks already weird